build/ci.go

```diff
@@ -87,44 +87,44 @@ var (
 	debExecutables = []debExecutable{
 		{
 			BinaryName:  "abigen",
-			Description: "Source code generator to convert Ethereum contract definitions into easy to use, compile-time type-safe Go packages.",
```

Comment:
Can we remove this file altogether? I don't think we use it at all. @hubchub can you confirm?

Reply:
https://github.com/dominant-strategies/go-quai/blob/main/Makefile#L12